blackboxopt/optimization_loops/dask_distributed.py

```diff
@@ -163,12 +163,12 @@ def run_optimization_loop(
                 break
 
         for evaluation in dask_scheduler.check_for_results(timeout_s=20):
-            optimizer.report_evaluation(evaluation)
+            optimizer.report(evaluation)
             evaluations.append(evaluation)
 
     while dask_scheduler.has_running_jobs():
         for evaluation in dask_scheduler.check_for_results(timeout_s=20):
-            optimizer.report_evaluation(evaluation)
+            optimizer.report(evaluation)
             evaluations.append(evaluation)
```

Comment:
Could this become the following? And if so, what do you prefer?
```python
    while dask_scheduler.has_running_jobs():
        new_evaluations = dask_scheduler.check_for_results(timeout_s=20)
        optimizer.report(new_evaluations)
        evaluations.extend(new_evaluations)
```

Reply:
Good catch! I think it should work and is like it more!